libraries/common/components/Swiper/index.jsx

```diff
@@ -0,0 +1,129 @@
+import 'react-id-swiper/src/styles/css/swiper.css';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import cls from 'classnames';
+import IDSwiper from 'react-id-swiper';
+import { Pagination, Navigation } from 'swiper/dist/js/swiper.esm';
+import SwiperItem from './components/SwiperItem';
+import { container, innerContainer, bullet, bulletActive } from './styles';
+
+/**
+ * The basic swiper component.
+ * @param {Object} props The component props.
+ * @returns {React.Node}
+ */
+function Swiper(props) {
+  const {
+    autoPlay,
+    interval,
+    children,
+    controls,
+    className,
+    classNames,
+    initialSlide,
+    rebuildOnUpdate,
+    slidesPerView,
+    maxIndicators,
+    indicators,
+    loop,
+    snapItems,
+    onSlideChange,
+  } = props;
+
+  const [swiper, setSwiper] = useState(null);
+
+  useEffect(() => {
+    if (swiper !== null) {
+      swiper.update();
+      swiper.on('slideChange', () => onSlideChange(swiper.realIndex));
+    }
+  }, [swiper]);
+
+  const useFraction = (maxIndicators && maxIndicators < children.length);
+  const paginationType = useFraction ? 'fraction' : 'bullets';
+  const el = (indicators && children.length > 1) ? '.swiper-pagination' : null;
+
+  const params = {
+    modules: [Pagination, Navigation], // Need to add Pagination, Navigation modules
+    containerClass: cls(innerContainer, classNames.container),
+    autoplay: autoPlay ? {
+      delay: interval,
+    } : false,
+    initialSlide,
+    navigation: !controls ? {
+      nextEl: null,
+      prevEl: null,
+    } : undefined,
+    pagination: {
+      el,
+      type: paginationType,
+      bulletClass: bullet,
```

Comment:
In the previous implementation it was possible to pass a `bulletClass` and a `bulletActiveClass` through the `classnames` prop. Now you apply local styling to them.

Reply:
One can now pass `bulletClass` and `bulletActiveClass` again.